src/commands/run.ts

```diff
@@ -63,7 +63,7 @@ export default class Run extends Command {
         preparedWorkflows = preparedWorkflows.filter(workflow => {
           const shouldInclude = workflows.includes(workflow.id)
           if (shouldInclude) {
-            workflow.sequences = workflow.sequences.filter(sequence => sequences.includes(sequence.id))
+            workflow.sequences = workflow.sequences.filter(sequence => sequences ? sequences.includes(sequence.id) : sequence)
```

Comment:
+1 for the need for pairwise testing. The interaction of flags.workflow and flags.sequence.